apps/src/applab/Visualization.jsx

```diff
@@ -20,6 +20,16 @@ var styles = {
   },
   phoneFrame: {
     marginBottom: 0
+  },
+  screenBlock: {
+    backgroundColor: 'rgba(0, 0, 0, 0.05)',
+    width: applabConstants.APP_WIDTH,
+    height: applabConstants.APP_HEIGHT - applabConstants.FOOTER_HEIGHT,
+    overflow: 'hidden',
+    zIndex: 4,
```

Comment:
May want to enforce, or at least call out with a comment, that this is `4` because:
```
  // layer 1 is design mode/div applab
  // layer 2 is items being dragged out from toolbox
  // layer 3 is overlay
```
(as recorded in applab/style.scss)